src/traces/waterfall/hover.js

```diff
@@ -37,19 +37,46 @@ module.exports = function hoverPoints(pointData, xval, yval, hovermode) {
 
     var size = (di.isSum) ? di.b + di.s : di.rawS;
 
-    if(!di.isSum) {
-        // format delta numbers:
-        if(size > 0) {
-            point.extraText = formatNumber(size) + ' ' + DIRSYMBOL.increasing;
-        } else if(size < 0) {
-            point.extraText = '(' + (formatNumber(-size)) + ') ' + DIRSYMBOL.decreasing;
-        } else {
-            return;
+    if(di.isSum) {
+        point.final = undefined;
+        point.initial = undefined;
+        point.delta = size - di.b;
```

Comment:
Hmm. I'm not sure we can call this a "delta" . `point.y` is probably best for the total bars.

Reply:
In respect to https://github.com/plotly/plotly.js/pull/3963#discussion_r293856811 displaying delta seems to be useful especially when there is a base; also when hovering using `hoverinfo`: `delta`.

Reply:
To make it similar to current behaviour, this option is dropped in https://github.com/plotly/plotly.js/pull/3963/commits/8432d8f770d912d90532f151d3abbf07e138ba23.